### DISCUSSION TITLE
Got the TypeError: network request failed, but I switch the network to the rinkeby testnet(neither mumbai  testnet） when I run the code locally

### DISCUSSION THREAD

**Discussion by Jackieyewang** [ethernet8023/vaportrade#30] [General]
The console error is: Access to XMLHttpRequest at 'https://cors.vaportrade.net/https://api.sequence.app/rpc/API/GetAuthToken' from origin 'http://localhost:3000' has been blocked by CORS policy:

**Comment by ethernet8023** [comment]
@Jackieyewang You'll need to change the CORS url in Vaportrade's Control Panel to a local instance of cors-everywhere.
